### PR TITLE
83572891_83572890

### PR DESCRIPTION
**Pull request by julpark-rh**
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83572891
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83572890

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ZCZQCC -> 83572891
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-R8ZH3P -> 83572890

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_system/mds_admin_daemon_socket_dump.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            if target_mds_name in mds["name"]:
+                mds_1 = mds["name"]
+        admin_daemon = (
+            f"cephadm shell ceph --admin-daemon /var/run/ceph/ceph-mds.{mds_1}.asok "
+        )
+        target_mds.exec_command(
+            sudo=True, cmd=f"{admin_daemon} config set debug_mds 10"
+        )
+        target_mds.exec_command(
+            sudo=True, cmd=f"{admin_daemon} config set debug_client 10"
```
Comment (hkadam134) [inline tests/cephfs/cephfs_system/mds_admin_daemon_socket_dump.py:91]: This config is not needed. Please remove this, Insteal you can have log_to_file and debug_ms=5.
And reset to defaults in the cleanup.

Comment (hkadam134) [reply tests/cephfs/cephfs_system/mds_admin_daemon_socket_dump.py:91]: Have these 2 
f"{admin_daemon} config set debug_ms 5"
f"{admin_daemon} config set log_to_file true"

Comment (hkadam134) [reply tests/cephfs/cephfs_system/mds_admin_daemon_socket_dump.py:91]: Fix the same in mds_admin_daemon_perf_flush.py script also

Comment (julpark-rh) [reply tests/cephfs/cephfs_system/mds_admin_daemon_socket_dump.py:91]: I have applied those fixes

**Review by AmarnatReddy — APPROVED** [review]

**Pull request merged**